tests/unit/DateTest.php

```diff
@@ -386,4 +386,34 @@ public function it_calculates_diff_correctly(): void
 
         $this->assertSame($expectedDiff->days, $diff->days);
     }
+
+    /**
+     * @return iterable<array{int}>
+     */
+    public function provideNumberOfDayOffsets(): iterable
+    {
+        yield [-10];
+        yield [-5];
+        yield [-1];
+        yield [0];
+        yield [1];
+        yield [5];
+        yield [10];
+    }
+
+    /**
+     * @test
+     * @dataProvider provideNumberOfDayOffsets
+     */
+    public function it_can_be_offset_by_days(int $days): void
+    {
+        $date = Date::fromYearMonthDay(2022, 5, 1);
+
+        $offsetDate = $date->offsetByDays($days);
+
+        $expectedDate = Date::fromDateTime($date->toDateTime()->modify($days < 0 ? "${days} days" : "+${days} days"));
```

Comment:
https://wiki.php.net/rfc/deprecate_dollar_brace_string_interpolation

Reply:
This is coming from the php-cs-fixer config for this library, happy to remove the braces from that ruleset in another PR?

Reply:
Can't use sprintf instead?

```suggestion
        $expectedDate = Date::fromDateTime($date->toDateTime()->modify(\sprintf('%s%d days', $days > 0 ? '+' : '', $days));
```

`+` is also redundant, so might just go with:
```suggestion
        $expectedDate = Date::fromDateTime($date->toDateTime()->modify(\sprintf('%d days', $days));
```

Reply:
Hadn't realised the + was redundant, let's just go with that then.